src/runtime-rs/crates/hypervisor/src/device/util.rs

```diff
@@ -69,9 +69,31 @@ pub(crate) fn get_virt_drive_name(mut index: i32) -> Result<String> {
     Ok(String::from(PREFIX) + std::str::from_utf8(&disk_letters)?)
 }
 
+pub fn do_increase_count(ref_count: &mut u64) -> Result<bool> {
```

Comment:
maybe add some document here to explain what does `true` or `false` returns means. 
Right now, it seems vague to me.

Reply:
fixed